.eslintrc.js

```diff
@@ -2,9 +2,16 @@ const eslintrc = require.resolve('@box/frontend/eslint/eslintrc.js');
 
 module.exports = {
     extends: [eslintrc],
```

Comment:
there's a cypress eslint preset that adds some helpful stuff

Reply:
Yeah agreed, but we should bring it into the frontend config project, I think. This is just a temporary workaround to get support up as quickly as possible.

Reply:
Good point. I'll add the Cypress plugin here shortly.